### DISCUSSION TITLE
Do step updates need refreshing?

### DISCUSSION THREAD

**Discussion by Dazz563** [github/vscode-github-actions#208] [Q&A]
Hey peeps!

Are we supposed to constantly click the refresh icon to see the steps in the workflows update, would be nice to have them update automatically like they do on github? 

Maybe I'm missing a setting around this?

**Comment by KetchupOnMyKetchup** [comment]
Thank you for pointing this out! 

I see an interesting difference in behaviors between re-running and running a new workflow.

When I re-run a workflow, I see the steps progress:
![rerunworkflow](https://github.com/github/vscode-github-actions/assets/7976517/a8912f2a-c297-4193-9990-a8c185678d46)

When I run a new workflow, I have to hit refresh to see it:
![runnewworkflow](https://github.com/github/vscode-github-actions/assets/7976517/5bed6ef5-2f00-4bd9-8834-08a579c7dbf1)

I will log an issue here: 
- https://github.com/github/vscode-github-actions/issues/229

Please feel free to edit it or add any additional info or screenshots, thank you so much for letting us know!

**Reply by Dazz563** [reply]
Awesome! Your description is way more advanced than mine 😂 but you nailed it! Thanks for the feedback, I dig the extension so if it helped then it's a win!

**Reply by KetchupOnMyKetchup** [reply]
Thanks so much for confirming that's what you saw! And, thank you as well for letting us know about this issue so we can log and prioritize it and hopefully make the app even better!! We appreciate the kind feedback about the extension!
